### ISSUE TITLE
[feature] Add decision symbol to geometric shape options 

### ISSUE THREAD

**Issue by beatrizoliveiira** [tldraw/tldraw#303]
Recently I used the app to write a flowchart to help me at work and I missed the triangle that has the functionality to show decisions, as a solution I just rotated the square to make it look like it. It would be interesting to add this new geometric shape to the app

![image](https://user-images.githubusercontent.com/65451957/142630289-e086b9db-2423-4ef1-9e77-9721d3428aca.png)

**Comment by steveruizok** [comment]
Any other shapes that we might want?

**Comment by mysticmind** [comment]
Line - draw a straight line.

**Comment by steveruizok** [comment]
I agree. The shape itself could still be an arrow, just the tool would create it without the arrowheads?

**Comment by proful** [comment]
Line Tool added as part of https://github.com/tldraw/tldraw/pull/343

Any other shape are we referring? Adding different shapes is a broader discussion going on here https://github.com/tldraw/tldraw/issues/360

**Comment by steveruizok** [comment]
Want to try adding a diamond?

**Comment by Kidlike** [comment]
Hi, thanks for this awesome tool, could we get a diamond (rhombus) shape please?

As the OP described, this is crucial for flowcharts, and it's one of the hardest shapes to do properly by hand.

---

PS: thanks for this awesome piece of software... It works really well on my Surface Pro 8, and even it automatically switches to the eraser if I use my pen from the back side / flipped (which is the intended purpose).

PS2: It also integrates very well inside StandardNotes with this plugin: https://github.com/antonheitz/sn-whiteboard.

**Comment by alex-kim-dev** [comment]
@Kidlike the next version has more shapes already https://beta.tldraw.com/

**Comment by Kidlike** [comment]
> @Kidlike the next version has more shapes already https://beta.tldraw.com/

Thanks for that, but there is still no diamond shape :(
